students/12th/najieun/westagram/js/login.js

```diff
@@ -0,0 +1,17 @@
+const button = document.getElementsByClassName('loginBtn')[0];
+const input = document.getElementsByClassName('input')[0];
+const name = document.getElementsByClassName("inputName")[0];
+const pw = document.getElementsByClassName("inputPw")[0];
+
+function changeColor() {
+    if(name.value && pw.value) {
+        button.removeAttribute('disabled');
+        button.style.backgroundColor='#0095F6';
+    } 
+}
+
+input.addEventListener('keyup', changeColor);
+
+button.onclick=function() {
+    window.location.href="../main.html";
+}
```

Comment:
이렇게도 할 수 있지만 위에서 input에 이벤트 거셨던 것처럼 하시는 방법도 있습니다!
어느 방법으로 하셔도 상관 없지만 일관성 있게 해주세요!
```suggestion
button.addEventListener('click', function() {
    window.location.href="../main.html";
}
```

Reply:
addEventListener로 통일했습니다~~